lib/topological_inventory/persister/workflow.rb

```diff
@@ -141,6 +144,22 @@ def send_changes_to_queue!
         )
       end
 
+      def send_task_updates_to_queue!
+        tasks_ic = persister.inventory_collections.select { |x| x.name == :service_instance_tasks }.first
+        if tasks_ic
```

Comment:
minor but unless there is some cleanup code at the end of the method (even then that can just be under an `ensure`) we usually prefer to `return if tasks_ic.nil?` rather than indenting the entire method in under a conditional block.